packages/grafana-toolkit/src/cli/utils/githubRelease.ts

```diff
@@ -0,0 +1,74 @@
+import { execLine } from './execLine';
+import { getPluginId } from '../../config/utils/getPluginId';
+import { getPluginJson } from '../../config/utils/pluginValidation';
+import { getCiFolder } from '../../plugins/env';
+import path = require('path');
+
+const ghrPlatform = (): string => {
+  switch (process.platform) {
+    case 'win32':
+      return 'windows';
+    case 'darwin':
+      return 'darwin';
+    case 'linux':
+      return 'linux';
+    default:
+      return process.platform;
+  }
+};
+
+const GHR_VERSION = '0.13.0';
+const GHR_ARCH = process.arch === 'x64' ? 'amd64' : '386';
+const GHR_PLATFORM = ghrPlatform();
+const GHR_EXTENSION = process.platform === 'linux' ? 'tar.gz' : 'zip';
+const PUBLISH_DIR = path.resolve(getCiFolder(), 'packages');
```

Comment:
Can all this be moved into the beginning of `private getGhr() {`

Reply:
except for the PUBLISH_DIR ... that can be in release()

Reply:
Roger doger